xbmc/video/VideoDatabase.cpp

```diff
@@ -4566,6 +4589,13 @@ void CVideoDatabase::UpdateTables(int iVersion)
       }
     }
   }
+  if (iVersion < 94)
```

Comment:
Does it really need to be one bigger? It's working fine this way. Is there another reason?